src/tv2_afvd_showstyle/helpers/pieces/jingle.ts

```diff
@@ -49,47 +51,35 @@ export function EvaluateJingle(
 	}
 
 	if (adlib) {
-		const p = GetJinglePartProperties(context, config, part)
-
-		if (JSON.stringify(p) === JSON.stringify({})) {
-			context.notifyUserWarning(`Could not create adlib for ${parsedCue.clip}`)
-			return
-		}
-
-		const props = p as Pick<
-			IBlueprintPart,
-			'autoNext' | 'expectedDuration' | 'prerollDuration' | 'autoNextOverlap' | 'disableOutTransition'
-		>
-
-		adlibPieces.push(
-			literal<IBlueprintAdLibPiece>({
-				_rank: rank ?? 0,
-				externalId: `${part.externalId}-JINGLE-adlib`,
-				name: effekt ? `EFFEKT ${parsedCue.clip}` : parsedCue.clip,
-				sourceLayerId: SourceLayer.PgmJingle,
-				outputLayerId: SharedOutputLayers.JINGLE,
-				lifespan: PieceLifespan.WithinPart,
-				metaData: literal<PieceMetaData>({
-					transition: {
-						isJingle: !effekt,
-						isEffekt: !!effekt
```

Comment:
Shouldn't `effekt` be stored somewhere and used in `executeActionSelectJingle` to generate similar metadata? Currently in there `isJingle` is always true